students/piechowski.michal/lesson_04_unit_testing/fibonacci_numbers_tests.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+import unittest
+parent_dir_name = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+sys.path.append(parent_dir_name + "/lesson_03_functions")
+
+from fibonacci_numbers import fib
```

Comment:
Not possible to move this up because then import wouldnt' succeed. If you want I can do dummy copy of file from lesson3 and put it to lesson4 but I would suggest to ingore stickler message